### DISCUSSION TITLE
On the weightage of the following indicators in the final score?

### DISCUSSION THREAD

**Discussion by allenthreee** [larics/icuas24_competition#46] [Q&A] [answered]
As the title, could you kindly provide some clarity on the weightage of the following indicators in the final score? For example:

1. Fruit count
3. Flight time
4. Path length

**Comment by fpetric** [comment, marked as answer]
Fruit count 50% of the score, time and path length 25% each.

**Reply by tanzhengo** [reply]
Are the flight time and path length calculated by multiplying XXX meters by 0.25 and XXX seconds by 0.25 respectively?
